std/fs/expand_glob.ts

```diff
@@ -167,15 +169,13 @@ export async function* expandGlob(
   yield* currentMatches;
 }
 
-/**
- * Synchronous version of `expandGlob()`.
+/** Synchronous version of `expandGlob()`.
  *
- * Examples:
+ * Example:
  *
- *     for (const file of expandGlobSync("**\/*.ts")) {
- *       console.log(file);
- *     }
- */
+ *      for (const file of expandGlobSync("**\/*.ts")) {
+ *        console.log(file);
+ *      } */
```

Comment:
seems like the `*/` should be on the following line.

Reply:
Done.